### DISCUSSION TITLE
can we put logs from other folder than storage ?

### DISCUSSION THREAD

**Discussion by tbauer42** [opcodesio/log-viewer#162] [Q&A] [answered]
Hello everyone ! 
Can we put logs from other folder than storage ? 
For example can we show logs from /var/log/*folder* ? 
I try to put it with a symbolic link on storage/logs but it doesn't work...
Thanks for you time

**Comment by arukompas** [comment, marked as answer]
Hey @tbauer42  You can, just define an absolute path in the configuration.

**Reply by tbauer42** [reply]
hey thanks for your answer, in the config > log-viewer.php ?
i dont see where i have to put the path ..

**Reply by arukompas** [reply]
In the `config/log-viewer.php` like so:

```php
    'include_files' => [
        '*.log',
        // add additional paths for log files, like so:
        '/path/to/other/laravel/logs/*.log'
    ],
```

Keep in mind that only Laravel log files are supported at the moment. Everything else, like Apache/MySQL/etc, will display empty log files.

Hope this helps 😀

**Reply by tbauer42** [reply]
yes thanks !!
